debian/patches/revert-551f560d-cloud-config-after-snap-seeding.patch

```diff
@@ -67,7 +67,7 @@ Last-Update: 2024-02-14
  
```

Comment:
Noble doesn't need that because 551f560d already released to noble before noble originally published in version `24.1_6`. So no need for us to revert that after the fact.